### ISSUE TITLE
Instance of 'SQLAlchemy' has no 'Column' member false positive

### ISSUE THREAD

**Issue by eliott02022000** [PyCQA/pylint#2917]
```
#!/usr/bin/env python3

from flask import Flask
from flask_sqlalchemy import SQLAlchemy


app = Flask(__name__)
app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:////tmp/test.db'
db = SQLAlchemy(app)

app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False


@app.route('/')
def hello_world():
    return 'Hello, World!'


class User(db.Model):
    id = db.Column(db.Integer, primary_key=True)
    username = db.Column(db.String(80), unique=True, nullable=False)
    email = db.Column(db.String(120), unique=True, nullable=False)

    def __repr__(self):
        return '<User %r>' % self.username

```

### Steps to reproduce
1.
2.
3.

### pylint --version output
Instance of 'SQLAlchemy' has no 'Column' member

**Comment by elamje** [comment]
[Link to my issue](https://github.com/PyCQA/pylint/issues/2987)

**Comment by Avasam** [comment]
This does not happen to me, do you have the following stubs installed?
```py
pylint-flask-sqlalchemy
sqlalchemy-stubs
sqlalchemy2-stubs
```

**Comment by mbyrnepr2** [comment]
It doesn't happen to me either. Closing.
```python
pylint 2.16.0-dev
astroid 2.13.0-dev0
Python 3.10.4
```
